docs/src/osdk/guide/intel-tdx.md

```diff
@@ -32,7 +32,7 @@ Therefore, it is recommended to use a Docker image to deploy the environment.
 Run a TDX Docker container:
 
 ```bash
-docker run -it --privileged --network=host --device=/dev/kvm asterinas/osdk:0.9.4-tdx
+docker run -it --privileged --network=host --device=/dev/kvm asterinas/osdk:0.9.4
```

Comment:
The Docker version number has long been outdated. Could you fix this number manually here and also do the `tools/bump_version.sh` script?